docs/basics/101-109-rerun.rst

```diff
@@ -138,8 +138,7 @@ compares one state (a commit specified with ``-f``/``--from``,
 by default the latest change)
 and another state from the dataset's history (a commit specified with
 ``-t``/``--to``). Let's do a :command:`datalad diff` between the current state
-of the dataset and the previous commit (called "HEAD~1" in Git terminology):
-of the dataset and the previous commit (called "``HEAD~1``" in Git terminology):
+ of the dataset and the previous commit (called "``HEAD~1``" in Git terminology):
```

Comment:
```suggestion
of the dataset and the previous commit (called "``HEAD~1``" in Git terminology):
```

Reply:
sorry, I'm afraid you have to commit again @lisanmo :smile:  (I added a stray space in the previous suggestion -- this is an "indentation" that made the build fail, this second suggestions rectifies this. I don't have the permissions to do a change to something that comes from your fork of the repository, so you have to press the button)

Reply:
Ah, good to know! I was wondering if I had to do that or if you could do it on your own.